build-aux/m4/bitcoin_qt.m4

```diff
@@ -284,22 +282,22 @@ AC_DEFUN([_BITCOIN_QT_IS_STATIC],[
     ])
 ])
 
-dnl Internal. Check if the link-requirements for static plugins are met.
+dnl Internal. Check if the link-requirements for a static plugin are met.
+dnl
+dnl _BITCOIN_QT_CHECK_STATIC_PLUGIN(PLUGIN, LIBRARIES)
+dnl --------------------------------------------------
+dnl
 dnl Requires: INCLUDES and LIBS must be populated as necessary.
 dnl Inputs: $1: A series of Q_IMPORT_PLUGIN().
```

Comment:
In 3184e52dab0ac4da32d4c4469d686765aa500446. Inputs is no longer a series of `Q_IMPORT_PLUGIN()`

Reply:
Thanks! [Fixed](https://github.com/bitcoin/bitcoin/pull/21363#issuecomment-791210965).